cinder/volume/drivers/vmware/volumeops.py

```diff
@@ -477,6 +481,7 @@ def _in_maintenance(self, summary):
                                                'inMaintenance']
         return False
 
+    @cached(cache=TTLCache(maxsize=128, ttl=300))
```

Comment:
This should be safe for hosts, but doesn't take cross-cluster migration into account for instances. If we put instances into this, I feel like this cannot be cached.